src/eth/Web3Service.js

```diff
@@ -224,8 +224,8 @@ export default class Web3Service extends PrivateService {
   }
 
   _isStillConnected() {
-    return promisify(this._web3.version.getNetwork)()
```

Comment:
This was the issue for ethers not listening to blocks. Would automatically disconnect as it returns undefined in the new implementation.